cats/src/scala/land/vola/cats/SparkTasks.scala

```diff
@@ -0,0 +1,14 @@
+package volaland
+package cats
+
+import _root_.cats.Id
+import _root_.cats.data.Kleisli
+import org.apache.spark.SparkContext
+
+object SparkTask {
+  def apply[A](f: SparkContext => A): SparkTask[A] =
+    Kleisli[Id, SparkContext, A](f)
```

Comment:
Why not use SparkTask[A] here?

Reply:
I did't understand proposal, it's standard apply factory (typed, not companion object). Where exactly do you want type?

Reply:
Nvm. My mistake